tabpy/tabpy_server/app/app.py

```diff
@@ -121,6 +134,9 @@ def _create_tornado_web_app(self):
                   default_filename="index.html")),
         ], debug=False, **self.settings)
 
+        signal.signal(signal.SIGINT, application.signal_handler)
```

Comment:
What happens on Windows?

Reply:
Tested on Windows, works as expected (not because of this specific line however).